src/sugar3/graphics/radiotoolbutton.py

```diff
@@ -102,12 +141,21 @@ def get_icon_name(self):
                                  getter=get_icon_name)
 
     def set_xo_color(self, xo_color):
+        '''
+        Sets xocolor
+        
+        Args:
+            xo_color (sugar3.graphics.xocolor): xocolor to be set
```

Comment:
sugar3.graphics.xocolor is the module, you want to add XoColor to refer to the class